generators/generator-constants.mjs

```diff
@@ -41,7 +41,7 @@ export const javaDependencies = {
    * hibernate version should match the one managed by https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-dependencies/SPRING_BOOT_VERSION
    * Required due to hibernate-jpamodelgen annotation processor.
    */
-  hibernate: '6.3.1.Final',
+  hibernate: '6.4.0.Final',
```

Comment:
We use hibernate from spring-boot/jhipster-bom. IFAIK this version is only used for `hibernate-jpamodelgen` dependency.

Reply:
Good to know. Thanks!